docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,177 @@
+ WebUI Instructions
+********************
+
+#. As an admin user, either create a new workspace or edit an existing workspace.
+
+#. In the "Namespace Bindings" section, enter a name for the namespace or leave it blank. If you
+   specify a namespace in the namespace field, the workspace is bound to that namespace. If the
+   field is left blank, the workspace is bound to the namespace specified in the
+   ``resource_manager.default_namespace`` section of your master configuration (when set), and is
+   otherwise bound to the default Kubernetes "default" namespace.
+
+#. Toggle the "Auto Create Namespace" option on or off. When enabled, the system automatically
+   creates a namespace in the cluster, allowing you to edit the resource quota directly in
+   Determined.
+
+#. Enter a resource quota (e.g., 5). This quota is editable only when "Auto Create Namespace" is on.
+   If the quota for a namespace is created in Kubernetes, it will appear here but won't be editable.
+   Determined displays the lowest quota on the namespace. If this lowest quota is a Determined-
+   created quota, you can edit it in the WebUI. If the lowest quota was created in Kubernetes, it
+   will not be editable in the WebUI.
+
+When completed, save your changes to apply the namespace bindings and resource quotas. Any submitted
+workloads that would cause workspace resource use to exceed the defined quota will stay pending
+until resources are available or the quota is increased.
+
+.. note::
+
+   The resource quotas set and displayed in Determined specifically apply to Kubernetes GPU request
+   limits. You can cap the GPU requests placed on a given workspace. Other Kubernetes resources like
+   CPU or memory request limits are not managed by Determined.
+
+.. important::
+
+   If a job is pending due to lack of resources (not resource quota) and you change the resource
+   quota, the job will be scheduled regardless of any new, more restrictive resource quota.
+
+Configure Additional Resource Managers
+======================================
+
+For each additional resource manager, configure the resource quota as needed. Each workspace in a
+cluster automatically has a default binding to its resource manager's configured or default
+namespace.
+
+******************
+ CLI Instructions
+******************
+
+You can use the :ref:`command-line interface (CLI) <cli-ug>` to set one workspace namespace binding
+at a time. To get help for setting workspace namespace bindings, run the following command:
+
+.. code:: bash
+
+   det w bindings -h
+
+Create a Workspace with Bindings and Quotas
+===========================================
+
+You can add bindings and set quotas during workspace creation using the following command where
+``cluster-name`` is the ``resource_manager.cluster_name`` specified in the master configuration:
+
+.. code:: bash
+
+   det w create <workspace-name> --cluster-name <cluster-name> --namespace <namespace-name> --resource-quota <resource-quota>
```

Comment:
`det w create <workspace-name> --cluster-name <cluster-name> --namespace <namespace-name>`

Reply:
We can't set the resource quota on workspaces bound to namespaces that weren't auto-created by Determined

Reply:
fix in #9720